lms/envs/common.py

```diff
@@ -1328,6 +1328,20 @@
         ],
         'output_filename': 'css/lms-style-xmodule-annotations.css',
     },
+    'style-certificates': {
+        'source_filenames': [
+            'sass/certificates/main-ltr.css',
```

Comment:
Woah, the Sass should not get compiled! @feanil, I thought this set up was needed for production/cache-busting. We're still just pointing to the already compiled CSS file in a way that our templates and various envs know about it. Sound right?